apps/blockchain/test/blockchain/account/repo/cache_test.exs

```diff
@@ -249,6 +249,34 @@ defmodule Blockchain.Account.Repo.CacheTest do
       assert found_code == code
     end
 
+    test "does not update account if account in cache is clean" do
+      ets_db = MerklePatriciaTree.Test.random_ets_db()
+      state = MerklePatriciaTree.Trie.new(ets_db)
+      address = <<1>>
+
+      # we need to create a blank account with storage
+      state_with_account = Account.reset_account(state, address)
```

Comment:
How would you feel about removing the above comment in favor of a different variable name? Something like:

```suggestion
      blank_account_with_storage = Account.reset_account(state, address)
```

Reply:
good idea. I change it in my next PR